contracts/PartyBid.sol

```diff
@@ -28,6 +28,14 @@ contract PartyBid is Party {
     //   (1) PartyStatus.ACTIVE on deploy
     //   (2) PartyStatus.WON or PartyStatus.LOST on finalize()
 
+    enum ExpireCapability {
```

Comment:
thoughts now that we have merged the repos about moving this to Party.sol? if we don't fully deprecate the shared contract, i think it would make the most sense to keep any shared capability in there. especially now because we are adding additional helper functions like `canExpire`, adding the enum, etc. - would be better to have them all conform to this interface for the expiration capability

Reply:
Lets get this merged and then we can do that.  I think its a good idea, but would rather merge and think how it might affect partybuy later